lib/absinthe/subscription.ex

```diff
@@ -179,6 +179,7 @@ defmodule Absinthe.Subscription do
 
       {doc_id, doc}
     end)
+    |> Map.new()
```

Comment:
You should simply replace the `Enum.map/2` prior to this with a `Map.new/2` call.

Reply:
Ahhh, nice catch! That would save some unnecessary processing.

Reply:
I did the dedup before this enum map: https://github.com/absinthe-graphql/absinthe/pull/1249/commits/2ee47cdc0102a802385e652407b4bac7c2bc665c

Reply:
I see that and I think that's generally fine but since order isn't important here `Map.new` is going to involve several less traversals. I'll show what I mean.

Reply:
Ahhh gotcha! Let me fix that

Reply:
https://github.com/absinthe-graphql/absinthe/pull/1249/commits/97ea4dc1ca699c47547496412b7d443e52092533